doc/developer/roadmap.rst

```diff
@@ -2,3 +2,115 @@
 Roadmap
 =======
 
+The roadmap is intended for larger, fundamental changes to
+the project that are likely to take months or years of developer time.
+Smaller-scoped items will continue to be tracked on our issue tracker.
+
+The scope of these improvements means that these changes may be
+controversial, are likely to involve significant discussion
+among the core development team, and may require the creation
+of one or more :ref:`nxep`.
+
+Installation
+------------
+
+We aim to make NetworkX as easy to install as possible.
+Some of our dependencies (e.g., graphviz and gdal) can be tricky to install.
+Other of our dependencies are easy to install on the CPython platform, but
+may be more involved on other platforms such as PyPy.
+Addressing these installation issues may involve working with the external projects.
+
+Sustainability
+--------------
+
+We aim to reduce barriers to contribution, speed up pull request (PR) review,
+onboard new maintainers, and attract new developers to ensure the long-term
+sustainability of NetworkX.
+
+This includes:
+
+- improving continuous integration
```

Comment:
Not sure if it belong in this document, but something about being present at workshops at conferences / doing outreach and "marketing" may be relevant here.

Reply:
Establish a good venue for users to ask questions and get help?

Reply:
I agree, but not sure it belongs here either.